template/template/android/app/build.gradle

```diff
@@ -49,6 +49,9 @@ react {
     //
     //   The list of flags to pass to the Hermes compiler. By default is "-O", "-output-source-map"
     // hermesFlags = ["-O", "-output-source-map"]
+
+    /* Autolinking */
+    autolinkLibrariesWithApp()
```

Comment:
nit: Those changes are unrelated from the linked PR @blakef (https://github.com/facebook/react-native/pull/44990)

Reply:
Yea, my script has failed me here.